### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-* Rename 'grade-a' and 'grade-c' to 'support-modern' and 'support-basic'
+## 0.11.0 / 2021-06-03
+* Add "declaration-property-unit-disallowed-list" with `px` for `font-size` and `line-height` (Volker E)
+* build: Update dependencies/devDependencies to latest (Ed Sanders)
+* build: Rename 'grade-a' and 'grade-c' to 'support-modern' and 'support-basic' (Volker E)
```
Comment (jdforrester) [inline CHANGELOG.md:6]: This isn't a mere build artefact, it's a major breaking change.

Comment (edg2s) [reply CHANGELOG.md:6]: Oh, we should have aliases for the old ones

Comment (edg2s) [reply CHANGELOG.md:6]: See #141

**Review by jdforrester — APPROVED** [review]

**Comment by jdforrester** [comment]
Have tweaked; leaving to @edg2s to decide if he's OK with my changes.

**Pull request merged**